keps/sig-instrumentation/3077-contextual-logging/kep.yaml

```diff
@@ -0,0 +1,38 @@
+title: Contextual logging
+kep-number: 3077
+authors:
+  - "@pohly"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-scheduling
+status: implementable
+creation-date: 2021-12-06
+reviewers:
+  - "@alculquicondor"
+  - "@serathius"
```

Comment:
Add @liggitt ?

Reply:
That would be useful as he already commented earlier, added. I just hope he has time.